Nodejs/Product/Nodejs/Intellisense/VsProjectAnalyzer.cs

```diff
@@ -769,7 +769,7 @@ sealed partial class VsProjectAnalyzer : IDisposable {
             }
 #endif
 
-            if (_implicitProject) {
+            if (_implicitProject && _analysisLevel != AnalysisLevel.None) {
```

Comment:
Moving this check closer to the Enqueue below after chatting with Dino, as not executing _projectFiles.TryRemove could potentially keep us from cleaning things up.